plugins/scaffolder/alpha-api-report.md

```diff
@@ -35,6 +35,9 @@ export type NextRouterProps = {
     TemplateListPageComponent?: React_2.ComponentType<TemplateListPageProps>;
     TemplateWizardPageComponent?: React_2.ComponentType<TemplateWizardPageProps>;
   };
+  logs?: {
+    autoOpen?: boolean;
+  };
```

Comment:
Hm. This is so far "up" in the interfaces and explicit. Feels like this could either have been a new app-config parameter, or making the log renderer part of `components` so you could give it another one with tweaked parameters (and letting it handle open/closed state internally) or something like that. @benjdlambert any opinions on what the interface should be here?

Reply:
I'd really prefer not to expose this so far up. As a compromise, would it be possible to store this state in LocalStorage? so that if you've opened the logs before it will keep them open for every job? We've also thought about making the buttons in the stepper actually open the logs too when you click on the step?

Reply:
It will solve the problem partially. 
For the person who first time opens the template they even not aware that you can open logs. 
Somehow these 3 dots are overlooked by most of people, unless you explicitly point to it. 

Maybe then indeed it can be passed via app-config parameter?

Reply:
@acierto still thinking about this a little more - it's unfortunate to have an `app-config.yaml` parameter too to be honest, but maybe it's the best approach right now. Wonder if any of the frontend system helps with this @Rugvip @freben?

Reply:
Hmm, is this really something that varies across setups that needs to be configured? Is there an underlying UX issues that should be fixed instead, making it more clear that logs are available?

Reply:
I wonder why logs are not shown right away. As it used to be in the previous version of the scaffolder template.

Reply:
They're not shown straight away based on user feedback that they don't provide value unless a job fails (happy to revisit this again if there's more requests for this to be the default behaviour) and with the move to supporting different renders of `TemplateOutputs` template authors and action creators should be using outputs to render actionable content rather than log lines with outputs.

Could we look at the logs button being somewhere else rather than in the context menu?

Reply:
Bogdan, Istvan and I had a small discussion this morning about this topic. We indeed want to fix the UX for this since the button for showing logs is hard to find. We will make it more prominent and obvious for users to find it.

Reply:
Don't pay too much attention to our design how it visually looks, but more to the solution for the problem we are discussing. Perhaps we can place a secondary button 'show logs' next to the other two buttons (cancel and start over)?
<img width="762" alt="image" src="https://github.com/backstage/backstage/assets/146729409/7ea95fd7-cafc-42e6-a3f5-a15fd3ddce03">

Reply:
Thanks for suggestion @lolsem. What do you think about this layout @benjdlambert?

Reply:
Just adding on a little here, I was the one that added the button bar and things around this, but my thinking for when I had more time was to add a Scaffolder section to the User Settings area where you could set your Org default and then a user can override them from there as needed. Doesn't totally solve this though as they would be global

Reply:
@acierto think that it's looking pretty good - happy to move it there if you'd prefer to highlight these actions there rather than the context menu.

Reply:
Oh indeed, missed that part to recreate with a proper change